code/style.css

```diff
@@ -1,4 +1,233 @@
 html {
-  background: #0025ff;
-  color: #fff;
+  background: #fffefe;
+  color: rgb(0, 0, 0);
+  font-family: "Montserrat", sans-serif;
+}
+
+body {
+  margin: 0;
+  min-height: 200vh;
+}
+
+/* Header with logo and soc-med links in fullscreen and only "CNN" on small screen*/
+
+header {
+  margin: 1rem;
+}
+
+.CNN {
+  text-decoration: none;
+  text-align: center;
+  padding-left: 0.5rem;
+  font-size: 3.5em;
+  font-weight: 900;
+  text-shadow: 5px 5px rgb(194, 194, 194);
+  min-height: 4rem;
+}
+
+a {
+  color: black;
+  text-decoration: none;
+}
+
+/*Covid virus logo*/
+.logo-imgs {
+  display: none;
+  height: 6rem;
+}
+
+.logo {
+  height: 6rem;
+}
+
+.logo1 {
+  position: relative;
+}
+
+.logo2 {
+  position: absolute;
+  opacity: 0;
+}
+
+.logo2:hover {
+  animation: fadein 1s ease-in alternate infinite;
+}
+
+@keyframes fadein {
+  from { opacity: 0; }
+  to { opacity: 1; }
+}
+
+/*links to soc-med*/
+.soc-med {
+  display: none;  
+  grid-template-columns: repeat(4, 1fr);
+  align-items: center;
+  gap: 2rem;
+}
+
+.soc-icon:hover,
+.nav-links:hover,
+.xpand-button:hover {
+  color: rgb(167, 15, 15);
+}
+
+/* News ticker */
+.black-stripe {
+  background-color: #000000;
+  height: 1.5em;
+  color: white;
+  font-weight: 900;
+  line-height: 100%;
+  text-align: right;
+}
+
+/* Navbar with links to categories*/
+.navbar {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.nav-links {
+  font-size: min(2rem, 4vw);
+  padding: 0.5rem;
+  max-width: max-content;
+  font-weight: 500;
+}
+
+h1 {
+  font-size: 4rem;
+  text-align: center;
+  text-decoration: overline underline;
```

Comment:
so this is such a small thing so don't pay attention to it, but the lines adds up on each other on mobile media mode, though they do it better when I checked on my actual phone.